### PR TITLE
Update repo links: sferik/rails_admin → railsadminteam/rails_admin

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

**Comment by jdufresne** [comment]
The jruby tests seem a bit flaky and fail intermittently.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 [![SemVer](https://api.dependabot.com/badges/compatibility_score?dependency-name=rails_admin&package-manager=bundler&version-scheme=semver)][semver]
 
 [gem]: https://rubygems.org/gems/rails_admin
 [ghactions]: https://github.com/railsadminteam/rails_admin/actions
-[coveralls]: https://coveralls.io/r/sferik/rails_admin
-[inch]: http://inch-ci.org/github/sferik/rails_admin
-[codeclimate]: https://codeclimate.com/github/sferik/rails_admin
+[coveralls]: https://coveralls.io/r/railsadminteam/rails_admin
+[inch]: http://inch-ci.org/github/railsadminteam/rails_admin
+[codeclimate]: https://codeclimate.com/github/railsadminteam/rails_admin
```
Comment (codealchemy) [inline README.md:14]: This currently returns a 404, maybe something needs to be updated in codeclimate as well? (the old link still works)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+[![Code Climate](https://codeclimate.com/github/railsadminteam/rails_admin.svg)][codeclimate]
 [![SemVer](https://api.dependabot.com/badges/compatibility_score?dependency-name=rails_admin&package-manager=bundler&version-scheme=semver)][semver]
 
 [gem]: https://rubygems.org/gems/rails_admin
 [ghactions]: https://github.com/railsadminteam/rails_admin/actions
-[coveralls]: https://coveralls.io/r/sferik/rails_admin
-[inch]: http://inch-ci.org/github/sferik/rails_admin
-[codeclimate]: https://codeclimate.com/github/sferik/rails_admin
+[coveralls]: https://coveralls.io/r/railsadminteam/rails_admin
+[inch]: http://inch-ci.org/github/railsadminteam/rails_admin
```
Comment (codealchemy) [inline README.md:13]: This is also down (and the badge returns a 404).

**Comment by jdufresne** [comment]
Thanks for the review @codealchemy 

I have both those links for now as I think that is preferred to a 404 or out of date link. We can add them back once the services have been migrated to railsadminteam (something I cannot do).

**Review by codealchemy — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,17 +1,13 @@
 # RailsAdmin
 
 [![Gem Version](https://img.shields.io/gem/v/rails_admin.svg)][gem]
-[![Build Status](https://img.shields.io/github/workflow/status/sferik/rails_admin/Test)][ghactions]
-[![Coverage Status](https://img.shields.io/coveralls/sferik/rails_admin.svg)][coveralls]
-[![Inline docs](http://inch-ci.org/github/sferik/rails_admin.svg)][inch]
-[![Code Climate](https://codeclimate.com/github/sferik/rails_admin.svg)][codeclimate]
```
Comment (mshibuya) [inline README.md:7]: I have set up Code Climate for the new repo, please add it back.
https://codeclimate.com/github/railsadminteam/rails_admin

Comment (jdufresne) [reply README.md:7]: Done. Restored. Sorry it took me so long to respond.

**Comment by mshibuya** [comment]
Thanks ❤️

**Pull request merged**